src/api/public/apidocs-new/paths/my_notifications_id.yaml

```
@@ -0,0 +1,31 @@
+put:
+  summary: Toggle the read/unread state of a notification.
+  description: |
+    Toggle the boolean value of the `delivered` attribute of a notification, which
+    marks the notification as read/unread.
+  security:
+    - basic_authentication: []
+  parameters:
+    - in: path
+      name: id
+      required: true
+      schema:
+        type: int
+  responses:
+    '200':
+      $ref: '../components/responses/succeeded.yaml'
+    '403':
+      $ref: '../components/responses/unauthorized.yaml'
+    '404':
+      description: Not Found
+      content:
+        application/xml; charset=utf-8:
+          schema:
+            $ref: '../components/schemas/api_response.yaml'
+          examples:
+            Not Found:
+              value:
+                code: not_found
+                summary: Notification with id 213453 does not exist
```

With my suggested changes in the controller, this message will be slightly different:

```suggestion
                summary: Couldn't find Notification with 'id'=213453
```